pkg/analysis/passes/readme/readme_test.go

```diff
@@ -56,3 +56,20 @@ func TestReadmeValid(t *testing.T) {
 
 	require.Len(t, interceptor.Diagnostics, 0)
 }
+
+func TestWithComment(t *testing.T) {
```

Comment:
can you add a test for no error? (A readme without comments, there is probably one already in the testdata)

Reply:
it is take caren of in testValidReadme test. No error or warnings